ecommerce/views_test.py

```diff
@@ -709,3 +710,41 @@ def test_products_viewset_detail(user_drf_client, coupon_product_ids):
             instance=Product.objects.get(id=coupon_product_ids[0])
         ).data
     )
+
+
+def test_companies_viewset_list(user_drf_client):
+    """ Test that the CompanyViewSet returns all companies """
+    companies = CompanyFactory.create_batch(3)
+    response = user_drf_client.get(reverse("companies_api-list"))
+    assert response.status_code == status.HTTP_200_OK
+    companies_list = response.json()
+    assert {company.get("id") for company in companies_list} == {
```

Comment:
Might want an assertion that `len(companies) == len(companies_list)`

Reply:
I think this line accomplishes essentially the same thing, if there is a mismatch in length this assertion should fail:
```
    assert {company.get("id") for company in companies_list} == {
        company.id for company in companies
    }
```

Reply:
Assuming there aren't duplicates in the list, it would, that's probably a case that's not worth protecting against